components/forecast/WeatherTab.tsx

```diff
@@ -52,14 +55,19 @@ export const WeatherTab: React.FC<WeatherTabProps> = ({zone, center_id, date}) =
 
   if (isLoading) {
     return (
-      <Center style={StyleSheet.absoluteFillObject}>
-        <ActivityIndicator />
+      <Center style={{flex: 1}}>
+        <VStack space={8} style={{flex: 1}} alignItems={'center'}>
+          <Body>Loading current weather forecast for the {zone.name} zone...</Body>
+          <ActivityIndicator />
+        </VStack>
       </Center>
     );
   } else if (isError) {
     return (
-      <Center style={StyleSheet.absoluteFillObject}>
-        <Body>Could not fetch weather forecast for zone {zone.name}</Body>
+      <Center style={{flex: 1}}>
+        <Body>
+          Could not fetch weather forecast for the {zone.name} zone: {forecastError?.message}
```

Comment:
Is the error here human-readable, or is it programmer-readable? (My suspicion would be it's the latter)

More information is not always helpful, especially when the user can't meaningfully act on it. It's often more reassuring to say something like "it's our fault, please try again".

We should probably get a coherent view from design on these topics.

Reply:
Yeah, I would love to have some more clarity here - this was pretty necessary for me to iterate locally, but I totally agree with you that the error is useless for users. Suiting both cases should be possible.

Reply:
(hope it's ok if we leave it as-is for now).

(also, I really, really dislike watching e.g. my girlfriend struggle with Apple's very opaque error modals for printer setup, when my Linux box showed me the exact error and I was able to debug it in seconds)